Source/Project64/User Interface/Notification Class.h

```diff
@@ -35,6 +35,8 @@ class CNotification :
 	//User Feedback
 	virtual void DisplayMessage(int DisplayTime, const wchar_t * Message) const;
 	virtual void DisplayMessage(int DisplayTime, LanguageStringID StringID) const;
+	virtual void DisplayMessageWide(int DisplayTime, const wchar_t * Message, ...) const;

```

Comment:
the problem with the current code uses stdstr_f to format, which is converting from utf16 to utf8/ansi and converting back which does not seem to work to well. We probaly need a stdwstr_f to be able to format strings in pure wchar_t